packages/x-charts/src/SparkLineChart/SparkLineChart.tsx

```diff
@@ -27,15 +22,21 @@ export interface SparkLineChartSlots
     LinePlotSlots,
     MarkPlotSlots,
     LineHighlightPlotSlots,
-    Omit<BarPlotSlots, 'barLabel'>,
-    ChartsTooltipSlots<'line' | 'bar'> {}
+    Omit<BarPlotSlots, 'barLabel'> {
+  /**
+   * Custom component for the tooltip popper.
+   * @default ChartsTooltipRoot
+   */
+  tooltip?: React.ElementType<ChartsTooltipProps>;
```

Comment:
This and the slotprops below seem to be used in multiple places, should we have a specific type for them?

Reply:
Yes, why not creating a `ChartsTooltipSlots` and `ChartsTooltipSlotProps` to avoid copy pasting.

The reason why I did it that was is that `XxxxSlots` is the typing of `slots` passed to the `Xxxx` component. But now `slots.tooltip` is not passed to the `ChartsTooltip` but to the `SparklineChart`

Reply:
I think it would still be useful to have them. It would likely simplify our lives, and if anyone needs to know if/how they are actually passed to the `ChartsTooltip` they can just check that